input/fsh/VS_Radiotherapy.fsh

```diff
@@ -42,13 +42,16 @@ Description: "Codes describing the techniques of teleradiotherapy (external beam
 * SCT#441799006 "Intensity modulated radiation therapy (procedure)"
 * SCT#1156530009 "Volumetric Modulated Arc Therapy (procedure)"
 // 3D term is not optimal since it refers to the planning procedure, not the therapy itself
+// * SCT#Requested (USCRS-352183)"Three Dimensional Teleradiotherapy (procedure)"
 * SCT#118641002 "Three dimensional treatment planning for external beam radiation therapy (procedure)"
 * SCT#1156526006 "Two dimensional external beam radiation therapy (procedure)"
 * SCT#168524008 "Radiotherapy - intraoperative control (procedure)"
 * SCT#1156529004 "External beam radiation therapy using particle passive scattering technique (procedure)"
 * SCT#1156528007 "External beam radiation therapy using particle spot scanning technique (procedure)"
 * SCT#1156525005 "Mixed beam external beam radiation therapy (procedure)"
 * SCT#169317000 "Neutron capture therapy (procedure)"
+* SCT#1156524009 "Intensity modulated external beam neutron radiation therapy (procedure)"
```

Comment:
Not sure why you are adding 1156524009.  Doesn't the modality of teleradiotherapy neutrons (80347004) plus the technique of IMRT (441799006) capture 1156524009?

Reply:
We requested a code, so I probably inferred from that that we wanted the code.

There is a note in the spreadsheet tracking code requests:  "Did we decide to delete IMNT - IM Neutron Therapy from the Teleradiotherapy Technique VS? We were granted a code for it: 1156524009 "

Reply:
CODEX RTTD group decided to drop the IMNRT term.

Reply:
Meant to add comment, not start a review....